### DISCUSSION TITLE
command not working in lua script in cheat

### DISCUSSION THREAD

**Discussion by Alestarov** [YimMenu/YimMenu#1847] [Q&A] [answered]
Please help who knows. I'm trying to write my script for Yim Menu, but I'm having a problem. When I try to change the value of "MP0_MONEY_EARN_BETTING" to 0 nothing happens. Before that, I created scripts in the Kiddions modest menu and everything worked there.

![Снимок экрана 2023-07-25 232434](https://github.com/YimMenu/YimMenu/assets/108485130/ae8cbad9-affa-41e1-9387-dc1e7e3d0315) 
![image](https://github.com/YimMenu/YimMenu/assets/108485130/dcdd4805-e03d-471b-b597-721ec6bd34f1)

**Comment by xiaoxiao921** [comment]
```lua
STATS.STAT_SET_INT("MP0_MONEY_EARN_BETTING", 0, true)
```

**Reply by Alestarov** [reply]
I tried this way, but it doesn't work(

**Reply by Alestarov** [reply]
![image](https://github.com/YimMenu/YimMenu/assets/108485130/3b3323de-9578-4af1-833a-864d1e7fa49a)

**Reply by Alestarov** [reply]
![image](https://github.com/YimMenu/YimMenu/assets/108485130/357b5c28-0a18-429e-86b4-be31eeaa82ae)

**Reply by maybegreat48** [reply]
Lua is case-sensitive

**Reply by MRVAST** [reply]
I tried to write in CapsLock, but is don't work
Well, I'll try again

**Reply by Alestarov** [reply]
Don't work(

**Reply by xiaoxiao921** [reply]
```lua
AlmenuS:add_button("Stat 0", function()
    script.run_in_fiber(function (script)
        STATS.STAT_SET_INT("MP0_MONEY_EARN_BETTING", 0, true)
    end)
end)
```

**Reply by Alestarov** [reply]
I don't understand why, but it doesn't work. Although it does not give an error


![Снимок экрана 2023-07-26 154432](https://github.com/YimMenu/YimMenu/assets/108485130/c7ef0e8d-6261-49a7-8923-d0c206615b1e)

**Reply by xiaoxiao921** [reply]
`MP0_MONEY_EARN_BETTING`

`MP1_MONEY_EARN_BETTING`

MP0 for first online character

MP1 for second online character

**Reply by xiaoxiao921** [reply, marked as answer]
ha nevermind, it need to be joaated too

```lua
AlmenuS:add_button("Stat 0", function()
    script.run_in_fiber(function (script)
        STATS.STAT_SET_INT(joaat("MP0_MONEY_EARN_BETTING"), 0, true)
    end)
end)
```

**Reply by Alestarov** [reply]
Thank you very much, everything works!!!!!!!!!
